ROLES.md

```diff
@@ -186,34 +152,37 @@ status is scoped to a part of the codebase.
 The following apply to the part of the codebase for which one would be an
 approver in an OWNERS file:
 
-- Reviewer of the codebase for at least 3 months.
+- Add yourself as a `writer` for the repositories you are responsible for
+  in [knative.yaml](../main/peribolos/knative.yaml) and/or
+  [knative-sandbox.yaml](../main/peribolos/knative-sandbox.yaml). After
+  this you should be an `approver` in the OWNERS file.
 
-- Should have reviewed or contributed to substantial PRs to the codebase.
+- Reviewer of the codebase for at least 3 months
```

Comment:
Do we want / need these specific time-based thresholds, or are contribution thresholds sufficient?

Reply:
Contribution thresholds are nice, but as the project matures and the rate of change slows, would it make sense to consider active involvement over a period of time. I am in favor of removing numeric contribution thresholds altogether, and feel that active and committed involvement in the project over time might be a better measure for these hurdles.

Reply:
This is carried over from the current document: https://github.com/knative/community/blob/main/ROLES.md#requirements-1

Reply:
The historical context here is that these were copied from Istio as a bulwark against the project membership and control changing too rapidly and against the desires of the originating companies.  I don't think we have the same concerns in the CNCF.

Reply:
I tend to prefer to leave the timing in because it serves as a guideline for people and reminds us to think about promoting people who have been involved for this amount of time. It also serves as a guideline for members to think about when they should try to move to the next level. However, I am going to propose changing it to "about 3 months" to give you some flexibility.